### PR TITLE
New Rule: Callback Phishing via extensionless rfc822 attachment

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

**Comment by morriscode** [comment]
This is pretty specific and gated akin to the other callback phishing messages. I would bet this is safe to ship.

**Review by bfilar — APPROVED** [review]

File: detection-rules/callback_phishing_extensionless_rfc822_image.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+name: "Callback Phishing via extensionless rfc822 attachment "
+description: "This rule detects messages with an unknown file_type (extensionless) and a content_type of 'message/rfc822' that do not contain an email, but rather an image file with Callback Phishing indicators."
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(attachments,
+          .file_type == "unknown"
+          and .content_type == "message/rfc822"
+          and not any(file.explode(.),
```
Comment (jkamdjou) [inline detection-rules/callback_phishing_extensionless_rfc822_image.yml:10]: is this `not` supposed to be here?

Comment (morriscode) [reply detection-rules/callback_phishing_extensionless_rfc822_image.yml:10]: yep

Comment (morriscode) [reply detection-rules/callback_phishing_extensionless_rfc822_image.yml:10]: I can move it to the line below if that makes it more clear
doesn't change the functionality

Comment (morriscode) [reply detection-rules/callback_phishing_extensionless_rfc822_image.yml]: that should be better?

**Pull request merged**